src/screens/Assistant/ResultItem.tsx

```diff
@@ -109,63 +115,120 @@ const ResultItem: React.FC<ResultItemProps & AccessibilityProps> = ({
   if (!tripPattern?.legs?.length) return null;
 
   return (
-    <TouchableOpacity
-      style={{paddingVertical: 4}}
-      onPress={onDetailsPressed}
-      hitSlop={insets.symmetric(8, 16)}
-      accessibilityHint={t(AssistantTexts.results.resultItem.a11yHint)}
-      accessibilityValue={{text: screenReaderSummary(tripPattern, t)}}
-      {...props}
-    >
-      <View style={styles.result}>
-        <ResultItemHeader tripPattern={tripPattern} />
-
-        <View style={styles.detailsContainer}>
+    <View style={styles.result} {...props}>
+      <ResultItemHeader tripPattern={tripPattern} />
+      <View style={styles.scrollContainer}>
+        <ScrollView
+          horizontal={true}
+          showsHorizontalScrollIndicator={false}
+          hitSlop={insets.symmetric(12, 20)}
+          contentContainerStyle={styles.detailsContainer}
+          accessibilityValue={{text: tripSummary(tripPattern, t)}}
+        >
           {tripPattern.legs.map(function (leg, i) {
-            if (leg.mode === 'foot') {
-              return (
-                <FootLeg
-                  key={leg.fromPlace.latitude}
-                  leg={leg}
-                  nextLeg={tripPattern.legs[i + 1]}
-                />
+            const legOutput =
+              leg.mode === 'foot' ? (
+                <FootLeg leg={leg} nextLeg={tripPattern.legs[i + 1]} />
+              ) : (
+                <TransportationLeg leg={leg} />
               );
-            }
-            return <TransportationLeg key={leg.serviceJourney.id} leg={leg} />;
+            return (
+              <View
+                style={styles.legOutput}
+                key={leg.serviceJourney?.id ?? leg.fromPlace.latitude}
+                {...screenReaderHidden}
```

Comment:
Note to self/thinking out loud. Wonder if we can make this as a component at some point

Reply:
Sure, a wrapper component might feel more seemless

Reply:
I think my main thing is that it will be more explicit and easier to document in a potential component library